### PR TITLE
Native Quick Start, Link Documentation, and Examples

### PR DESCRIPTION
**Pull request by tylermcginnis**

### CODE REVIEW

File: packages/react-router-dom/docs/api/Prompt.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # &lt;Prompt>
 
-TODO
+Re-exported from core [`Prompt`](../../../react-router/docs/api/Prompt.md)
```
Comment (tylermcginnis) [inline packages/react-router-dom/docs/api/Prompt.md]: @ryanflorence Can you confirm this is right? It's how all the other files were but it wasn't importing correctly for me. I'm assuming there's some build step I wasn't running.

File: packages/react-router-native/docs/guides/quick-start.md [hunk 2]
```diff
@@ -1,18 +1,137 @@
 # Quick Start
 
-Gimme 10 minutes, you'll be up and running.
-
-## Exponent
-
-Easiest way to get started is probably with Exponent.
+If this is your first time building a React Native app, we recommend you check out the official ["Getting Started"](https://facebook.github.io/react-native/docs/getting-started.html) guide.
```
Comment (mjackson) [inline packages/react-router-native/docs/guides/quick-start.md]: 👍

File: packages/react-router-native/index.ios.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { AppRegistry } from 'react-native'
 
-import App from './examples/BoringExample'
+import App from './examples/BasicExample'
```
Comment (mjackson) [inline packages/react-router-native/index.ios.js]: 😂

File: packages/react-router-dom/docs/api/Prompt.md (hunk 1 above)
Comment (mjackson) [inline packages/react-router-dom/docs/api/Prompt.md]: Yes, this is correct. `react-router-native` re-exports everything from core

https://github.com/ReactTraining/react-router/blob/a757d7f2c56bfbc54833a82620a67f8ddf83e7e7/packages/react-router-native/main.js#L1

**Pull request merged**